### DISCUSSION TITLE
xpore MinION data access

### DISCUSSION THREAD

**Discussion by sumin5784** [GoekeLab/xpore#61] [General]
Hello :)

I've been looking for HEK293T cell line nanopore sequencing data, and thank you for the sharing the data.

I have some questions about SRA accession IDs.

There are WT-rep1,2,3 and WT0/25/50/75/100-1,2,3 replicates in accession list. What are the differences between WT and WT0/25/50/75/100?
Also, could you inform me the differences between KO, KD, and KD-Ctrl samples?

I already downloaded HEK293T-WT-rep1,3 and HEK293T-KO-rep1,3.
WT-rep1 and KO-rep1 are both seemed correct.
But WT/KO-rep3 has 5~6 subfolders and each folder contains only several fast5 files. It seems like it has only a few reads, is there any specific reason for this?

Bests,
Sumin

**Comment by ploy-np** [comment]
Hi @sumin5784 ,

WT-25/50/75 are the mixture between WT and KO. For example, WT-25 means that there are 25% of WT and 75% of KO.
KO is the samples with Mettl3 knockout, KD is the samples with Mettl3 knockdown.
WT-0 is the same as KO, and WT-100 and KD-Ctrl are the same as WT. However, they are different datasets.

Can I ask where you downloaded the dataset?
